WcaOnRails/spec/controllers/registrations_controller_spec.rb

```diff
@@ -57,25 +57,23 @@
     it "can delete registration" do
       expect(RegistrationsMailer).to receive(:notify_registrant_of_deleted_registration).with(registration).and_call_original
 
-      expect do
-        delete :destroy, id: registration.id
-      end.to change { ActionMailer::Base.deliveries.length }.by(1)
+      delete :destroy, id: registration.id
 
       expect(flash[:success]).to eq "Deleted registration and emailed #{registration.email}"
-      expect(Registration.find_by_id(registration.id)).to eq nil
+      expect(Registration.find_by_id(registration.id).deleted?).to eq true
     end
 
     it "can delete multiple registrations" do
       registration2 = FactoryGirl.create(:registration, competition: competition)
 
       expect(RegistrationsMailer).to receive(:notify_registrant_of_deleted_registration).with(registration).and_call_original
       expect(RegistrationsMailer).to receive(:notify_registrant_of_deleted_registration).with(registration2).and_call_original
-      expect do
-        xhr :patch, :do_actions_for_selected, competition_id: competition.id, registrations_action: "delete-selected",
-                                              selected_registrations: ["registration-#{registration.id}", "registration-#{registration2.id}"]
-      end.to change { ActionMailer::Base.deliveries.length }.by(2)
```

Comment:
Instead of deleting this, we can check for enqueued jobs (you can see an example of this above). Something like this:

```ruby
       end.to change { enqueued_jobs.size }.by(1)
```